docs/data-sources/aviatrix_network_domains.md

```diff
@@ -0,0 +1,38 @@
+---
+subcategory: "TGW Orchestrator"
+layout: "aviatrix"
+page_title: "Aviatrix: aviatrix_network_domains"
+description: |-
+  Get a list of all Network Domains and detail
+---
+
+# aviatrix_all_network_domains
+
+The **aviatrix_network_domains** data source provides details about all Network Domains created by the Aviatrix Controller. Available as of provider version 2.23+.
+
+## Example Usage
+
+ ```hcl
+ # Aviatrix All Network Domains Data Source
+ data "aviatrix_network_domains" "foo" {
+   
+ }
+ ```
+
+
+## Attribute Reference
+
+Use above data source, the following attributes are exported:
```

Comment:
This could be reworded. I think you should just have the second part: `The following attributes are exported:`

Reply:
have fixed